packages/opentelemetry-api-metrics/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/api-metrics",
-  "version": "0.33.0",
+  "version": "1.9.0",
```

Comment:
should we move the api into the api directory ?

Reply:
> Should we make metrics API its own top level directory or "sink" the tracing api by one level in the directory structure?

I though we were saying that we wanted the api to be top level of the repo ? Don't we want to merge both trace and metrics api ?

Reply:
We talked about this in the SIG and decided to keep them separate in order to keep bundle size down for web